### PR TITLE
Added Getting Started for Service Registry

### PR DESCRIPTION
**Pull request by bibryam**

### CODE REVIEW

File: getting-started-service-registry/README.adoc [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-long: Red Hat Openshift Service Registry
```
Comment (smccarthy-ie) [inline getting-started-service-registry/README.adoc:14]: Suggest: 
:product-long: Red Hat OpenShift Service Registry

File: getting-started-service-registry/README.adoc [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+:product: Service Registry
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://cloud.redhat.com/beta/application-services/sr/
```
Comment (smccarthy-ie) [inline getting-started-service-registry/README.adoc:17]: Suggest: 
:service-url: https://cloud.redhat.com/beta/application-services/service-registry/

(The beta URL changed this week. https://cloud.redhat.com/beta/application-services/sr/ gives a 404 - see https://issues.redhat.com/browse/MGDSTRM-5144)

File: getting-started-service-registry/README.adoc [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As a developer of applications and services, you can use {product-long} to create and set up Service Registry instances and connect your applications and services to these instances. {product} is a managed cloud service that enables you to manage schema and API definitions in your applications without having to install, configure, run, and maintain your own Service Registry clusters.
+
+//For more overview information about {product}, see [variablized link to overview here https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/].
```
Comment (smccarthy-ie) [inline getting-started-service-registry/README.adoc:36]: SR portal doc link will be https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/

File: getting-started-service-registry/README.adoc [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+[id="proc-creating-service-account_{context}"]
+== Creating a service account to connect to a Service Registry instance in {product}
+
+To connect your applications or services to a Service Registry instance in the {product} web console, you need to create a service account, copy and save the generated credentials, and copy and save the instance URL. You'll use the service account information later when you configure your application.
+
+.Prerequisites
+* You've created a Service Registry instance and the instance is in *Ready* state.
+
+.Procedure
+. In the *Service Registry Instances* page of the web console, for the relevant Service Registry instance that you want to connect to, select the options icon (three vertical dots) and click *View connection information*.
```
Comment (smccarthy-ie) [inline getting-started-service-registry/README.adoc:100]: Suggest: 
In the *Service Registry* instances page...

File: getting-started-service-registry/README.adoc [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+== Uploading a schema to {product}
+
+After you create a Service Registry instance, you can upload a schema and start serializing/deserializing messages with the schema.
+
+.Prerequisites
+* You've created a Service Registry instance and the instance is in *Ready* state.
+
+.Procedure
+. In the *Service Registry Instances* page of the web console, select the name of the Service Registry instance that you want to upload a schema to.
+. Click *Upload Artifact* and fill up the form to define the schema details. Click *Upload* to complete the operation.
```
Comment (smccarthy-ie) [inline getting-started-service-registry/README.adoc:139]: Suggest:
In the **Service Registry** instances page...

Comment (smccarthy-ie) [inline getting-started-service-registry/README.adoc:139]: Suggest moving "Click Upload to complete the operation" to a separate step after the description of each field below.

File: getting-started-service-registry/README.adoc [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+* Is the new schema in the Artifacts table?
+endif::[]
+ifndef::qs[]
+* Verify that the new schema is listed in the Artifacts table.
+endif::[]
+
+[role="_additional-resources"]
+== Additional resources
+* https://kafka.apache.org/081/documentation.html#configuration[Configuration^] in Kafka
+(TODO) add link the RHOSR docs
```
Comment (smccarthy-ie) [inline getting-started-service-registry/README.adoc:189]: Link to the RHOSR docs will be https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry

**Comment by smccarthy-ie** [comment]
@bibryam Looks great, thanks! Left some initial comments. When this PR is merged, I will follow up with a separate PR to clean up the content/formatting and test reuse for doc portal and console

@pmuir @EricWittman Can you do a technical review of this PR? Thanks!

File: getting-started-service-registry/README.adoc (hunk 1 above)
Comment (smccarthy-ie) [reply getting-started-service-registry/README.adoc:14]: Attributes are auto-generated and need to be added separately in https://github.com/redhat-developer/app-services-guides/blob/main/_artifacts/document-attributes.adoc. Will do this in separate PR...

**Comment by smccarthy-ie** [comment]
Thanks again @bibryam. It's OK with @pmuir and @EricWittmann to merge this initial draft to get the ball rolling...

**Comment by pmuir** [comment]
@smccarthy-ie please don't merge PRs that cause the build to fail!

**Pull request merged**